features/_bfpxe/test/capabilities.d/capabilities.list

```diff
@@ -1,2 +1 @@
-/usr/bin/arping cap_net_raw=ep
-/usr/bin/ping cap_net_raw=ep
\ No newline at end of file
+
```

Comment:
If this file is empty anyway, we can just rm it (along with the `capabilities.d` directory). The test itself (`test_capabilities.py`) must remain however, as it also checks capabilities on `/usr`, `/var`, etc...